Mathlib/Tactic/CC/Addition.lean

```diff
@@ -1622,25 +1622,25 @@ def removeParents (e : Expr) (parentsToPropagate : Array Expr := #[]) : CCM (Arr
       if pocc.symmTable then
         let some (rel, lhs, rhs) ← p.relSidesIfSymm? | failure
         let k' ← mkSymmCongruencesKey lhs rhs
-        if let some lst := (← get).symmCongruences.find? k' then
+        if let some lst := (← get).symmCongruences[k']? then
           let k := (p, rel)
           let newLst ← lst.filterM fun k₂ => (!·) <$> compareSymm k k₂
           if !newLst.isEmpty then
             modify fun ccs =>
               { ccs with symmCongruences := ccs.symmCongruences.insert k' newLst }
           else
             modify fun ccs =>
-              { ccs with symmCongruences := ccs.symmCongruences.erase k' }
+              { ccs with symmCongruences := ccs.symmCongruences.remove k' }
```

Comment:
leanprover/lean4#4725 is now merged, so this will change back to `erase` with the next nightly.